module/testmodule/module.json

```diff
@@ -13,6 +13,24 @@
 			"model": "rdk:test:helper"
 		}
 	],
+	"models": [
```

Comment:
This is a robot config, and `models` is not a valid field. Would you mind making a new `module/testmodule/expected_meta.json` file to compare the `meta.json` models against?